gravitee-apim-console-webui/src/management/settings/client-registration-providers/client-registration-provider/client-registration-provider.component.ts

```diff
@@ -48,6 +50,18 @@ export class ClientRegistrationProviderComponent implements OnInit, OnDestroy {
 
   public formInitialValues: unknown;
 
+  public trustStoreTypes = [
```

Comment:
What do you think about requesting path or cntent immediately with this select?

`Java Trust Store (.jks) with path`
`Java Trust Store (.jks) with content`